docs/dx.rst

```diff
@@ -0,0 +1,246 @@
+.. _dx:
+
+DNAnexus
+========
+
+DNAnexus paths on stor are prefixed with ``dx://`` and have two components:
+``dx://<PROJECT>:<FILE_OR_FOLDER>``
+where project and file can be virtual paths (i.e., human names) or canonical paths
+(opaque globally unique IDs that the platform assigns) - see below for more details.
+
+Canonical Paths on DNAnexus
+---------------------------
+Every file resource on DNAnexus is given a canonical ID to refer it by. This is
+internally given by the platform and is not determined by stor. However, stor
```

Comment:
can you shorten this paragraph.  It should be 1 sentence like: `Files on DNAnexus have a globally unique immutable handle (called a `dxid`) and also a virtual path in each project.` rather than this whole paragraph.

Reply:
Fixed